Rubrik/Private/Submit-Request.ps1

```diff
@@ -42,8 +42,11 @@ function Submit-Request {
                 # Delete operations (and some post) generally have no response body, skip JSON formatting and store the response from Invoke-WebRequest
                 if (Test-PowerShellSix) {
                     # Uses the improved ConvertFrom-Json cmdlet as provided in PowerShell 6.1
+                    # In the case of DELETE, there is no content (json body) to parse.
                     $result = if ($null -ne ($WebResult = Invoke-RubrikWebRequest -Uri $uri -Headers $header -Method $method -Body $body)) {
-                        ConvertFrom-Json -InputObject $WebResult.Content
+                        if ($null -ne $WebResult.Content) {
+                            ConvertFrom-Json -InputObject $WebResult.Content
```

Comment:
$WebResult.Content is $null here, and trying to parse will result in an exception thrown from ConvertFrom-Json.

```
ConvertFrom-Json $foo.Content
ConvertFrom-Json: Cannot convert 'System.Byte[]' to the type 'System.String' required by parameter 'InputObject'. Specified method is not supported.
```